plugins/filter/reproducer_gerrit_infix.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+    name: reproducer_gerrit_infix
+    short_description: Maps a hostname to the infix needed for cloning a repo
+    for that hostname.
+    description:
+        - Maps a hostname to the infix needed for cloning a repo from that
+          hostname.Some gerrit instances add an infix like "gerrit", or "r"
+          to the repo clone url.
+    options:
+        _input:
+            description:
+                - repo hostname where we want to clone from.
+            type: str
+            required: true
+"""
+
+EXAMPLES = """
+    - name: Get infix to clone repo from rdo gerrit
```

Comment:
i thought you were making up new words for a minute there :)
TIL "infix" ... I knew about prefix and suffix but had not heard of this word before